### PR TITLE
Google fit gawen

### LINKED ISSUE #40: Add Google_fit_activity
Added google_fit_activity function to excavator.

### PR DESCRIPTION
**Pull request by GawenMonnot**
fixes #40

### CODE REVIEW

File: src/activities/google/google_fit_activity.rs [hunk 1]
```diff
@@ -0,0 +1,64 @@
...
+    pub duration         : String,
+    pub startTime        : String,
+    pub endTime          : String,
+    pub fitnessActivity  : String
+}
+
+#[allow(non_snake_case)]
+impl Fit {
+    pub fn saveToDb(&self, conn: &Connection) -> Result<(), rusqlite::Error> {
+        //for elem in self.fit.iter() {
```
Comment (chansuke) [inline src/activities/google/google_fit_activity.rs:17]: delete this comment out, thanks

File: src/activities/google/google_fit_activity.rs [hunk 2]
```diff
@@ -0,0 +1,64 @@
...
+                    &my_uuid.to_string(),
+                    fitnessActivity,
+                    startTime,
+                    endTime,
+                    duration
+                ],
+            )
+            .map_err(|err| println!("{:?}", err))
+            .ok();
+       // }
```
Comment (chansuke) [inline src/activities/google/google_fit_activity.rs]: And this line too

File: src/activities/google/google_fit_activity.rs (hunk 1 above)
Comment (chansuke) [reply src/activities/google/google_fit_activity.rs:17]: Thanks

**Review by chansuke — APPROVED** [review]
LGTM. Thanks

**Review by chansuke — CHANGES_REQUESTED** [review]
@GawenMonnot Sorry Gawen, can you format the code with this command
`cargo fmt --all`

**Comment by GawenMonnot** [comment]
> @GawenMonnot Sorry Gawen, can you format the code with this command
> `cargo format --all`

@chansuke Do you mean `cargo fmt --all` ?

**Comment by GawenMonnot** [comment]
> @GawenMonnot Sorry Gawen, can you format the code with this command
> `cargo fmt --all`

It should be done now.

**Review by chansuke — APPROVED** [review]
Everything is ready to merge. Thanks, Gawen!!!

**Pull request merged**